WMS 11825 - first version

MarcGueury opened the pull request:
# Add here your WMS 11825

# General requirements
1. Information in the workshop is adequate and updated
2. Code is correct and working
3. Links are correct
4. Make sure you enter the help email link in your manifest.json
5. Please make sure WMS URLs are updated as needed after your PR is approved


# Checklist - Refer to the QA document for the complete list 
Please confirm that the following is completed before submitting your PR

- [X] All filenames are lower case (including folders, images, files, etc.)
- [X] Filenames are descriptive
- [X] Your workshop folder structure should be similar to the one used in the sample workshop (https://github.com/oracle-livelabs/common/tree/main/sample-livelabs-templates/sample-workshop)
- [X] Are you using multiple versions (desktop/, sandbox/, tenancy/)? Make sure that each of them contains a manifest.json and an index.html
- [X] Image references in markdown contain an alternate text

carmenberdant1 reviewed (changes requested):
Hi @MarcGueury, for lab 4 please have the image names in lowercase and modify them in the digital-assistant.md file too.  
Thank you!

MarcGueury commented:
I did this:
- updated files of lab4 with all lowercase
- modified digital-assistant.md to reflect it
- sync this branch with main 

Thanks, please review.

MarcGueury commented:
@carmenberdant1
